packages/main/src/RPA/Hubspot.py

```diff
@@ -309,6 +475,83 @@ class Hubspot:
         deals = hs.search_for_objects("DEALS", "hs_lastmodifieddate", "GTE", yesterday)
         print(deals)
 
+    .. _batch-inputs:
+
+    Batch Inputs
+    ============
+
+    When retrieving information, the library automatically batches requests
+    that are provided as lists, see \`Get object\` keyword for an example,
+    but when wishing to create or update many objects, the library provides
+    a batching system.
+
+    In order to start a batch, you must first call the \`Create new batch\`
+    keyword. This initializes a new batch to accept inputs. If a batch
+    already exists when you call this keyword, it will be lost and a new
+    blank one will be started.
+
+    Once the batch has been initialized, you can add inputs one at a time with
+    \`Add input to batch\` or many at a time with \`Extend batch with inputs\`.
```

Comment:
```suggestion
    ``Add input to batch`` or many at a time with ``Extend batch with inputs``.
```
in REST docstrings (or in [release notes](https://rpaframework.org/releasenotes.html)) we usually use double '``' to highlight a piece of keyword/code. (as single '`' just emphasises it like single '*')

This way you won't need anymore to use repr `r"""...` docstrings anymore.

Reply:
By escaping the backtick, it actually invoked a special component of the Robot Framework docgen tool and it creates an inter keyword link, that's why I do it.